### ISSUE TITLE
Docs: Manifest incorrect in chapter `Deploying using a Manifest`

### ISSUE THREAD

**Issue by ghillert** [spring-cloud/spring-cloud-dataflow-server-cloudfoundry#425]
The example `manfifest.yml` in chapter `Deploying using a Manifest` is missing a colon and 2 single quotes:

This

```yaml
...
SPRING_APPLICATION_JSON {"maven": { "remote-repositories": { "repo1": { "url": "https://repo.spring.io/libs-release"} } } }
...
```

should be:

```yaml
...
SPRING_APPLICATION_JSON: '{"maven": { "remote-repositories": { "repo1": { "url": "https://repo.spring.io/libs-release"} } } }'
...
```

**Comment by sabbyanandan** [comment]
Review SCDF samples ref. guide since we add a note about it there explicitly.

**Comment by ilayaperumalg** [comment]
This was merged as a1b30eb
